src/imitation/scripts/expert_demos.py

```diff
@@ -179,6 +190,16 @@ def callback(locals_: dict, _) -> bool:
         output_dir = os.path.join(policy_dir, "final")
         serialize.save_stable_model(output_dir, policy, vec_normalize)
 
+      # Final evaluation of expert policy.
```

Comment:
Consider adding a TODO here so we don't forget to fix `VecNormalize` related issues.